toolchain/check/testdata/class/virtual_modifiers.carbon

```diff
@@ -15,6 +15,10 @@ package Modifiers;
 base class Base {
   virtual fn H();
 
+  // CHECK:STDERR: modifiers.carbon:[[@LINE+4]]:3: error: impl without base class [ImplWithoutBase]
+  // CHECK:STDERR:   impl fn I();
+  // CHECK:STDERR:   ^~~~~~~~~~~~
+  // CHECK:STDERR:
   impl fn I();
```

Comment:
Can you move the `impl fn` portions of this test out to a separate `// --- fail_modifiers.carbon` split-file within this test file?

(Background: we put passing and failing tests into different split-files within a test file. Split-files with errors should be named `fail_...` and ones without errors should not -- the test infrastructure checks this. The idea here is that if autoupdate causes a passing split-file to start failing or vice versa, we will catch that despite the autoupdate changing the expectations.)

Reply:
The tests seem to have got wedged somehow, but I'd expect they would fail for this reason.

Reply:
Ah, yeah - updated this test as requested and mucked about with the other test cases since there was some loss of coverage of the non-fail cases now that the requirements were a bit more strict, so pulled those out into more split-file chunks in this file. Let's see if the testing unwedges...